auth-server/src/services/auth.ts

```diff
@@ -49,6 +48,7 @@ export const getPwdResetTokenKey = (userId: number): string => `PRT-${userId}`;
 export const getAddressLoginKey = (address: string): string => `ALN-${address}`;
 export const getAddressSignupKey = (address: string): string => `ASU-${address}`;
 export const getSetCredentialsKey = (address: string): string => `SCR-${address}`;
+export const getEmailVerificationTokenKey = (token: string): string => `SCR-${token}`;
```

Comment:
Should it be called `EVT`? I think SCR is just a copy from above.

Reply:
fixed